packages/fxa-payments-server/src/components/PaymentForm/index.stories.tsx

```diff
@@ -179,4 +99,92 @@ const mockValidatorState = (): ValidatorState => ({
   },
 });
 
-init();
+type MostPaymentFormProps = Omit<PaymentFormProps, 'submitNonce'>;
+
+const defaultPaymentFormProps: MostPaymentFormProps = {
+  locale: 'auto',
+  inProgress: false,
+  confirm: false,
+  customer: undefined,
+  plan: PLAN,
+  onSubmit: action('onSubmit'),
+  onChange: () => {},
+  onMounted: () => {},
+  onEngaged: () => {},
+  getString: (x: string) => x,
+};
+
+const storyWithProps = (paymentFormProps: MostPaymentFormProps) => {
+  return () => {
+    const [submitNonce, refreshSubmitNonce] = useNonce();
+    const { locale } = paymentFormProps;
+    return (
+      <MockApp languages={[locale || 'auto']}>
+        <SignInLayout>
+          <div className="product-payment">
+            <button className="button" onClick={refreshSubmitNonce}>
+              Refresh submit nonce
+            </button>
+            <p>Current nonce: {submitNonce}</p>
+            <PaymentForm {...paymentFormProps} submitNonce={submitNonce} />
+          </div>
+        </SignInLayout>
+      </MockApp>
+    );
+  };
+};
+
+export const Default = storyWithProps(defaultPaymentFormProps);
+
+export const WithExistingCard = storyWithProps({
+  ...defaultPaymentFormProps,
+  customer: CUSTOMER,
+});
+
+export const WithoutPlan = storyWithProps({
```

Comment:
I noticed there wasn't a difference among the default, Without Plan, Without Confirmation, and All Invalid stories. Or is it something I'm missing? Should we still include these stories or need to open a ticket to fix this if this falls out of scope. We are going to open a ticket for a [similar issue for PlanDetails](https://github.com/mozilla/fxa/pull/14400#discussion_r1026726651).

Reply:
I've updated [this Jira tickets](https://mozilla-hub.atlassian.net/browse/FXA-6303) info to update the stories and remove them as part of that change.